markdown/ch1.markdown

```diff
@@ -495,22 +495,26 @@ warnings.filterwarnings('ignore', '.*Axes.*compatible.*tight_layout.*')
 
 > **A quick note on plotting {.callout}**
 >
-> The code above does a few neat things to make our plots
-> prettier. First, `%matplotlib inline` is a Jupyter notebook
-> [magic command](http://ipython.org/ipython-doc/dev/interactive/tutorial.html#magics-explained),
+> The code above does a few neat things to make our plots prettier.
+
+> First, `%matplotlib inline` is a Jupyter notebook [magic
+> command](http://ipython.org/ipython-doc/dev/interactive/tutorial.html#magics-explained),
 > that simply makes all plots appear in the notebook rather than pop up a new
-> window. Second, we import `matplotlib.pyplot` then direct it to use our own
-> plotting style `plt.style.use('style/elegant.mplstyle')`. You will see a block
-> of code like this before the first plot in every chapter.
+> window. If you are running a Jupyter notebook interactively, you can use
+> `%matplotlib notebook` instead to get an interactive figure, rathen than a
+> static image of each plot.
+>
+> Second, we import `matplotlib.pyplot` then direct it to use our own plotting
+> style `plt.style.use('style/elegant.mplstyle')`. You will see a block of code
+> like this before the first plot in every chapter.
```

Comment:
It shouldn't break anything? We could just comment that there's no need for them to do this, but their plots will look a little different.